src/stave.js

```diff
@@ -76,6 +76,10 @@ export class Stave extends Element {
     if (!this.formatted) this.format();
 
     this.start_x = x;
+    const begBarline = this.modifiers[0];
```

Comment:
I don't think `Stave` is guaranteed to have modifiers, and this will crash in that instance.

Maybe wrap this block in `if (this.modifiers.length > 0) { ... }`.